### PR TITLE
Fmwia

### PR DESCRIPTION
**Pull request by joakim-hove**
Implement summary keyword FMWIA - number of abondoned injectors.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp [hunk 1]
```diff
@@ -565,7 +565,9 @@ class Well {
     bool segmented_density_calculation() const { return true; }
     double alq_value() const;
     double temperature() const;
+    bool hasInjected( ) const;
     bool hasProduced( ) const;
+    bool updateHasInjected( );
     bool updateHasProduced();
     bool cmp_structure(const Well& other) const;
     bool operator==(const Well& data) const;
```
Comment (akva2) [inline opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp:573]: this needs updating.

Comment (joakim-hove) [reply opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp:573]: Thanks - I have updated the `operator==` - I assume that was what you meant?

Comment (akva2) [reply opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp:573]: yes, exactly.

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
Comments?

**Pull request merged**